head-and-mouth/Head and Mouth Cancer Datasets.ipynb

```diff
@@ -93,175 +118,173 @@
     "\n",
     "case_ids = list(map(lambda case: case['case_id'], result['data']['uiCase']))\n",
     "unique_case_ids = numpy.unique(case_ids)\n",
```

Comment:
`unique` should be prefixed with `np.`, not `numpy.` (since the package is imported with `import numpy as np`)

Reply:
I did change `import numpy` to `import numpy as np`, and forgot to change the prefix on codes. Thank you @turbomam!